inventory/allinone-ha.yaml

```diff
@@ -0,0 +1,160 @@
+all:
+  hosts:
+    bastion:
+      ansible_connection: local
+    allinone01:
+      ansible_host: 192.168.211.20
+    allinone02:
+      ansible_host: 192.168.211.174
+    zk03:
+      ansible_host: 192.168.211.151
```

Comment:
Could we make these vars so user can pass?

Reply:
this yaml file just like a config file for different deployment mode. and I have mentioned in README to check these before deployment. not only these IP addresses, there are some other options in this file also need to check before deployment.

sure yes, we can make these vars allow passing, but I think maybe it is can be implement in a wrapping script e.g. `deploy.sh` in future improvement.